src/components/pages/devices/devicesGrid/devicesGrid.js

```diff
@@ -31,15 +32,30 @@ export class DevicesGrid extends Component {
       deviceColumnDefs.lastConnection
     ];
 
-    // TODO: This is a temporary example implementation. Remove with a better version
     this.contextBtns = [
-      <Btn key="tag">Tag</Btn>,
-      <Btn key="schedule">Schedule</Btn>,
-      <Btn key="reconfigure">Reconfigure</Btn>,
-      <Btn key="delete" svg={svgs.trash} onClick={this.openDeleteFlyout}>{props.t('devices.flyouts.delete.apply')}</Btn>
+      <Btn key="jobs" svg={svgs.reconfigure} onClick={() => this.openFlyout('jobs')}>{props.t('devices.flyouts.jobs.title')}</Btn>,
+      <Btn key="delete" svg={svgs.trash} onClick={() => this.openFlyout('delete')}>{props.t('devices.flyouts.delete.title')}</Btn>
     ];
   }
 
+  openFlyout = (flyoutName) => this.setState({ openFlyoutName: flyoutName });
```

Comment:
openFlyout = (flyoutName) => () => this.setState({ openFlyoutName: flyoutName });